docs/UserGuide.md

```diff
@@ -72,38 +72,55 @@ Format: `help`
 
 ### Adding a residence: `add`
 
-Adds a new residence to the list of residences, default for CLEANING_STATUS is ‘cleaned’.
+Adds a new residence to the list of residences, default for clean status is ‘clean’. Valid clean status inputs are `y`,`n`, `clean` and `unclean`
 
-Format: ` add n/NAME_OF_APARTMENT a/ADDRESS [clean/[y or n]] [t/TAG] `
+* Names can include `@` inside. e.g `pinnacle@duxton`
+* Names can include numbers. e.g `Block71`
+
+Format: ` add n/NAME_OF_APARTMENT a/ADDRESS [c/VALID_CLEAN_STATUS] [t/TAG]... `
 
 Examples:
 * `add n/Melville Park a/22 Simei Street 1, #10-02, S529948`
-* `add n/Clementi HDB a/459A Clementi Ave 3, #04-257, S121459 clean/n`
+* `add n/Clementi HDB a/459A Clementi Ave 3, #04-257, S121459 c/n`
 
 ### Listing all residences: `list`
 
 Shows a list of all residences in the app.
 
 Format: `list`
 
+* The displayed list of residences is always sorted.
+* Unclean residences come before clean residences.
+
+### Listing residences with bookings starting in the next 7 days: `remind`
+
+Shows a list of all residences with bookings starting in the next 7 days.
+
+Format: `remind`
+
+* The displayed list of residences is always sorted.
+* Unclean residences come before clean residences.
+
 ### Editing a residence: `edit`
 
-Edits the cleaning status of an existing residence.
+Edits the given fields of an existing residence (excludes bookings, see `editb` instead to edit bookings).
 
-Format: `edit INDEX c/n `
+Format: `edit INDEX c/VALID_CLEAN_STATUS`
```

Comment:
i think can include all the optional fields of residences also? we are allowing them to edit those as well i think?